### PR TITLE
TRN-23: Completed assignment 1 where created form containing name

### PR DESCRIPTION
**Pull request by secretayush**
1st Assignment code review.

### CODE REVIEW

File: assin.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<body>
+<?php
+$f_name =  $_POST["fname"];       //store first name via post method
+$l_name =  $_POST["lname"];       //store last name via post method
+$full_name = $f_name." ".$l_name; //concatenate the fname and lname
+
+?>
+
+
+Hello <?php echo $full_name;?>   <!-- print the full name in web page -->
```
Comment (hemuvyas) [inline assin.php:15]: Do we need to close php tag above? Can't we do this without closing it.

File: assignment1.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  <h1>Form</h1>
+
+  <form action="assin.php" method="post">
+    <!-- using post method to send the data to assin.php file -->
+
+    First name : <input type="text" name="fname" id="fname"><br><br>
+
+    Last name : <input type="text" name="lname" id="lname"><br><br>
+
+    Full name : <input type="text" id="fullname" name="fullname" disabled><br><br>
```
Comment (hemuvyas) [inline assignment1.html:19]: Disabled attribute is not the best choice, change this with appropriate attribute. Also reply the difference between the disabled attribute and which you need to use here.

File: assin.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+<?php
+$f_name =  $_POST["fname"];       //store first name via post method
```
Comment (hemuvyas) [inline assin.php:8]: Add a check condition at the top level for checking if the method of form submission.
Also I can't see any check for User will input only alphabets, I believe we can add digits also in the field of first & last name. Kindly add that also.

File: assin.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+$l_name =  $_POST["lname"];       //store last name via post method
+$full_name = $f_name." ".$l_name; //concatenate the fname and lname
```
Comment (hemuvyas) [inline assin.php:10]: Why do we need to concat first name and last name if we have a field for full name already in the form?

**Review by hemuvyas — CHANGES_REQUESTED** [review]

File: assin.php (hunk 1 above)
Comment (secretayush) [reply assin.php:15]: Yes, we can do this without closing the first php code.

File: assignment1.html (hunk 2 above)
Comment (secretayush) [reply assignment1.html:19]: Disabled:
We can sent when the according form submits
elements do not receive focus.


Readonly:
We can't sent when the according form submits
elements can receive the focus.

**Comment by hemuvyas** [comment]
This PR is fine from my side, merging this.

**Pull request merged**